client/src/i18n/en/depot.json

```diff
@@ -1,22 +1,30 @@
 {
-    "DEPOT": {
-        "ADD_DEPOT": "Add a Depot",
-        "ALL_DEPOTS": "All Depots",
-        "CREATED":"Depot created with success",
-        "DELETED":"Depot deleted with success",
-        "DELETE":"Delete",
-        "DEPOTS_LIST": "Depots list",
-        "DESCRIPTION": "This module allows you to List, Create, Edit and Delete a Depot",
-        "EDIT_DEPOT": "Edit the information in a Depot",
-        "ENTITY" : "Depot",
-        "LABEL":"Depot",
-        "MAIN": {
-            "TITLE": "Depot Management"
-        },
-        "NO_DEPOT": "No Depot",
-        "NOT_ALLOWED_ACTIONS":"Protected Depot, you cannot perform modification on this depot",
-        "TITLE": "Depot Management",
-        "UPDATED":"Depot updated with success",
-        "WAREHOUSE":"Warehouse"
+  "DEPOT" : {
+    "ADD_DEPOT" : "Add a Depot",
+    "ALL_DEPOTS" : "All Depots",
+    "CREATED" : "Depot created with success",
+    "CHANGE" : "Change Depot",
+    "DELETED" : "Depot deleted with success",
+    "DELETE" : "Delete",
+    "DEPOTS_LIST" : "Depots list",
+    "DESCRIPTION" : "This module allows you to List, Create, Edit and Delete a Depot",
+    "EDIT_DEPOT" : "Edit the information in a Depot",
+    "ENTITY" : "Depot",
+    "LABEL" : "Depot",
+    "MAIN" : {
+      "TITLE" : "Depot Management"
+    },
+    "NO_DEPOT" : "No Depot",
+    "NOT_ALLOWED_ACTIONS" : "This depot is protect.  You cannot modify its properties.",
```

Comment:
Whoops, this should say "this depot is protected." instead of "this depot is protect."